packages/react/src/theme/components/button.ts

```diff
@@ -0,0 +1,130 @@
+import { defineStyleConfig } from "@chakra-ui/react";
+import { colors } from "../foundations";
+
+const config = defineStyleConfig({
+  baseStyle: ({ colorScheme }) => ({
+    fontFamily: "Mulish",
+    fontStyle: "normal",
+    fontWeight: "700",
+    fontSize: "18px",
+    lineHeight: "28p",
```

Comment:
Mistenker at det er en typo her.
```suggestion
    lineHeight: "28px",
```